### ISSUE TITLE
Is this production ready?

### ISSUE THREAD

**Issue by superdolt** [Wind2009-Louse/DBProject#1]

**Comment by Wind2009-Louse** [comment]
No, it's just for my course presentation. Neither i think it fully implement the paper's structure, nor do i want to improve it again.

**Comment by superdolt** [comment]
Does it work well? Any bugs so far? Pls provide the instructions to compile. Great work nonetheless. I can try it and see if any issues to help test etc. If u have a to do list, do mention

**Comment by Wind2009-Louse** [comment]
I think the basic functionality has been implemented, but without stress testing or anything like that. Therefore, it might be a bit buggy when it comes to big data. And I think to achieve the efficiency mentioned in report, C++-only is not enough, it might need a more underlying implementation. My project is C++-only to implement the basic principles, which is definitely very inefficient.

I used Visual Studio 2015 for Windows to code and compiled it with  MSVC, so there was no Makefile or anything like that (as mentioned in another issue).

Regarding the TODO list, since this is over a year old code, I've basically forgotten about it lol.

**Comment by superdolt** [comment]
Pls make a Make file.

Also, can u write this as c? Possible to negotiate a contract work for writing to c? Curious how much u charge. Do u have telegram account? Can discuss further on implementation

**Comment by Wind2009-Louse** [comment]
I have no interest in rebuilding it currently, and I don't use telegram either.
As for makefile, I will write it later.

**Comment by superdolt** [comment]
Thx in advance for the makefile

**Comment by superdolt** [comment]
How's makefile progress? Sorry anxious to try out
